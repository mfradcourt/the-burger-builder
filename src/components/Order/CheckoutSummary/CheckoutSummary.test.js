import React from 'react';
import { shallow } from "enzyme";

import CheckoutSummary from './CheckoutSummary';

describe('<CheckoutSummary />', () => {
  let wrapper;

  it('should render CheckoutSummary', () => {
    wrapper = shallow(
      <CheckoutSummary/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
