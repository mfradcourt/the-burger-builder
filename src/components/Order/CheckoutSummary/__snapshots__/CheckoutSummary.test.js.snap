// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckoutSummary /> should render CheckoutSummary 1`] = `
<div
  className="CheckoutSummary"
>
  <h1>
    We hope it tastes well!
  </h1>
  <div
    style={
      Object {
        "margin": "auto",
        "width": "100%",
      }
    }
  >
    <burger />
  </div>
  <button
    btnType="Danger"
  >
    CANCEL
  </button>
  <button
    btnType="Success"
  >
    CONTINUE
  </button>
</div>
`;
